content/en/about/board/2023-07-27.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+show_meta: false
+title: '2023-07-27'
+---
+
+### Date and Time of Meeting
+
+2023-07-27, 2pm - 3pm CEST
+
+* Call to order: 14:06 CEST
+* Adjourned: 15:01 CEST
+* Next Meeting: 2023-08-31, 7pm - 8pm CEST
```

Comment:
```suggestion
* Next Meeting: 2023-08-31, 9am CEST
```

@gruetter I have this in the calendar as our next meeting.

Also announcing the start time should do (saving us typing and potential mistakes).

Reply:
I was referring to[ this schedule](https://innersourcecommons.slack.com/archives/G012NHSA7DZ/p1685034149803599). Is this not the correct one, @spier ?

Reply:
@gruetter The slack thread that you linked to had contains discussion of the meeting times. However the first image in that thread are NOT the times that we ended up with. We discussed various options and eventually decided on two slots, one optimized for Asia+Europe, and one for Americas+Europe.

The correct schedule is what is in the "InnerSource Commons Calendar".

Reply:
@gruetter I am merging this, as I confirmed that 9am CEST is indeed the start time of the next meeting.